tests/spec/w3c/conformance-spec.js

```diff
@@ -62,4 +63,29 @@ describe("W3C — Conformance", () => {
     const doc = await makeRSDoc(ops);
     expect(doc.querySelectorAll("#conformance .rfc2119").length).toBe(0);
   });
+
+  it("allows conformance section to be completely overridden via .override css class", async () => {
+    const body = `
+      <section>
+        <h2>Plain section</h2>
+        <p>MUST SHOULD MAY.</p>
+      </section>
+      <section id="conformance" class="override">
```

Comment:
Too late but shouldn't this be more like an attribute, like `data-override`?

Reply:
I prefer using CSS classes for none “data” things, tbh.